tests/test_standard_tags.py

```diff
@@ -57,7 +58,7 @@ def test_url_with_query_string(self):
             value_as_regular_expression=True,
         )
 
-    def setup_url_with_sensitive_query_string(self):
+    def setup_url_with_sensitive_query_string_legacy(self):
```

Comment:
if we change the name, won't the manifest files all need to be updated ?

Reply:
no problem here, as we don't change class name.